workers/unity/Packages/com.improbable.gdk.core/Systems/CleanReactiveComponentsSystem.cs

```diff
@@ -24,19 +22,20 @@ public class CleanReactiveComponentsSystem : ComponentSystem
         protected override void OnCreateManager(int capacity)
         {
             base.OnCreateManager(capacity);
-
-            var worker = WorkerRegistry.GetWorkerForWorld(World);
-            view = worker.View;
-
             GenerateComponentGroups();
         }
 
         private void GenerateComponentGroups()
         {
+            // TODO: Remove workaround when UTY-936 is fixed
```

Comment:
I'd remove this. Especially as we have a better work around for that anyway that we will likely use soon to unblock IL2CPP. So we will likely remove all this code before that ticket.

Reply:
I'd still like to keep this to avoid this being forgotten about in case we dont get the time to do the alternative you have in mind.